Asynchronous JavaScript/script.js

```diff
@@ -0,0 +1,84 @@
+const admissionStatus = true;
+
+let candidateActivity;
+let studentActivity;
+let admissionStatusResponse;
+let candidateActivityResponse;
+let studentActivityResponse;
+
+const candidateActivities = [
+    "Check your admission status",
+    "While waiting to know your admission status, explore the courses taught in our college",
+    "Congratulations, you have been admitted"
+]
+
+const studentActivities = [
+    "The first two weeks of resumption is for orientation",
+    "Exam starts on 5th of September, 2031",
+    "First semester ends today, second semester starts in three weeks"
+]
+
+const errorMessages = [
+    "You have not been admitted",
+    "You have no application record with our college"
+]
+
+const setResponseTime = (activity, seconds) => {
+    return new Promise.resolve(
+        setTimeout( 
+            () => { 
+                console.log(activity)
+        }, seconds )
+    )
+}
+
+
+const candidateStatus = () => {
+    return new Promise(
+        (resolve, reject) => {
+            setTimeout(
+                () => {
+                    if (admissionStatus) {
+                            resolve(candidateActivities[2])
+                    } else {
+                        reject(new Error(errorMessages[0]))
+                    }
+        }, 7000)
+    })
+}
+
+
+const studentTimeline =  async function() {
+
+    candidateActivity = candidateActivities[0]
```

Comment:
Always remember to use `const` or `let` to declare your variables.
Like in this case, this should probably be `let candidateActivity = candidateActivities[0]`